### PR TITLE
1068107: CloudFoundry: fix for default path issue

### PR DESCRIPTION
**Pull request by lkillgore**

### CODE REVIEW

File: Extension/CloudFoundry/cloudfoundry.ts [hunk 1]
```diff
@@ -6,16 +6,17 @@ import path = require('path');
...
 var cfEndpointUrl = tl.getEndpointUrl(cfEndpoint, false);
 var cfEndpointAuth = tl.getEndpointAuthorization(cfEndpoint, false);
 var workingDir = tl.getInput('workingDirectory', true);
 
 var cfPath = tl.which('cf');
-var cfToolLocation = tl.getInput('cfToolLocation');
-if(cfToolLocation != tl.getVariable('System.DefaultWorkingDirectory')) {
-    //custom tool location for cf CLI was specified
-    cfPath = cfToolLocation;
+if(tl.filePathSupplied('cfToolLocation')) {
```
Comment (yacaovsnc) [inline Extension/CloudFoundry/cloudfoundry.ts]: p999: space after 'if'? :)

**Review by yacaovsnc — APPROVED** [review]

File: Extension/CloudFoundry/cloudfoundry.ts (hunk 1 above)
Comment (lkillgore) [reply Extension/CloudFoundry/cloudfoundry.ts]: I prefer that too, but I'd rather have it consistent with the rest of the file.

Comment (madhurig) [reply Extension/CloudFoundry/cloudfoundry.ts]: You can fix the entire file by using Shift + Option +F in VSCode on Mac :)

**Review by madhurig — APPROVED** [review]

File: Extension/CloudFoundry/cloudfoundry.ts (hunk 1 above)
Comment (lkillgore) [reply Extension/CloudFoundry/cloudfoundry.ts]: 👍

**Pull request merged**